model-optimizer/mo/utils/ir_reader/extenders/strided_slice_extender.py

```diff
@@ -12,8 +14,12 @@ class StridedSlice_extender(Extender):
 
     @staticmethod
     def extend(op: Node):
+        input_shape = op.in_port(0).data.get_shape()
         for attr in StridedSlice.get_mask_names():
-            Extender.attr_to_list(op, attr)
+            if op[attr] != '':
+                Extender.attr_to_list(op, attr)
```

Comment:
also can it be a case when `op` does not have attribute `new_axis_mask` (and the others) at all? According to the specification it is acceptable.

Reply:
I have changed the condition, now this case is handled